docs/calico-with-docker/without-docker-networking/README.md

```diff
@@ -167,6 +167,9 @@ See the [IPv6 worked example](IPv6.md) for a worked example.
 For details about advanced policy options read the 
 [Advanced Network Policy tutorial](../../AdvancedNetworkPolicy.md).
 
+## Make a container reachable from the Host-Interface (Internet)
+
+You cannot simply use `-p`on `docker run` to expose ports. We have a working example on how to [expose a container port to the internet](../../ExposePortsToInternet.md)
```

Comment:
I'd also like to see the first sentence reworded a bit.  As it is, it sounds a bit like Calico is difficult because you can't do this simple thing.

Could you change this to something like:

"Calico also allows you to expose ports on your host's interface to forward traffic from the host to your containers, similar to calling `docker run` with the `-p` flag."